tasks.py

```diff
@@ -383,7 +383,7 @@ def layer2_dev_env():
 # See dev-env/bgp/README.md for some more information.
 def bgp_dev_env(ip_family):
     dev_env_dir = os.getcwd() + "/dev-env/bgp"
-    frr_volume_dir = dev_env_dir + "/frr-volume"
+    frr_volume_dir = os.path.expanduser('~') + "/.metallb/frr-volume"
```

Comment:
I'd rather provide an alternative location via a parameter, than dropping the volume on an arbitrary location outside of the checkout

Reply:
ok, sure, will take alternative location via a parameter and by default let it use the current checkout directory.